src/Test/AbstractFixerTestCase.php

```diff
@@ -161,6 +165,12 @@ protected function doTest($expected, $input = null, \SplFileInfo $file = null, F
         Tokens::clearCache();
         $tokens = Tokens::fromCode($expected);
 
+        if (!$fixer->isCandidate($tokens)) {
+            $this->assertFalse($tokens->isChanged(), 'Fixer should not touch Tokens on candidate check.');
+
+            return;
```

Comment:
We need to `return` here. Since the passed code (`$tokens` at this point) is not a candidate for the fixer. It allows for testing the `isCandidate`-logic the same way we test code that is suitable for the fixer.
I'll do a little refactor to make it more clear :)